backend/src/main/java/ai/verta/modeldb/utils/ModelDBHibernateUtil.java

```diff
@@ -569,4 +578,61 @@ public static Query getWorkspaceEntityQuery(
     }
     return query;
   }
+
+  /**
+   * Don't change anything in this function because based on the function backend READY status will
+   * define. If you want to define new migration then add new if check for your migration in `if (migration) {` condition.
+   */
+  @SuppressWarnings("unchecked")
```

Comment:
We should follow the migration in background we did last time. We should have two flags were migration is triggered by one and we start using the rolebinding based on another

Reply:
This migration code will work in background. we will use the describe flag in the description to enable migration. but I do not understand about second flag.